nix/modules/scripts.nix

```
@@ -35,17 +35,86 @@
         set -xeuo pipefail
         trap "cd $PWD" EXIT
 
-        cd versions/0_1
-        nix flake update
-        cd ../../
+        export VERSIONS_DIR="./versions/''${1}"
+        export DEFAULT_VERSIONS_DIR="$(nix flake metadata --no-write-lock-file --json | jq --raw-output '.locks.nodes.versions.locked.path')"
 
-        nix flake lock --update-input versions --override-input versions "path:./versions/0_1"
+        (
+          cd "$VERSIONS_DIR"
+          nix flake update --tarball-ttl 0
```

try out this for an atomic update procedure. it's unclear how this affects clients, as they don't have access to the path

```suggestion
          nix flake update --tarball-ttl 0 --override-input holochain-flake ../..
```